examples/lib/stories/parallax/sandbox_layer.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/parallax.dart';
+import 'package:flutter/painting.dart';
+
+class SandBoxLayerParallaxGame extends BaseGame {
+  final Vector2 planeSpeed;
+  final ImageRepeat planeRepeat;
+  final LayerFill planeFill;
+  final Alignment planeAlignment;
+
+  SandBoxLayerParallaxGame({
+    required this.planeSpeed,
+    required this.planeRepeat,
+    required this.planeFill,
+    required this.planeAlignment,
+  });
+
+  @override
+  Future<void> onLoad() async {
+    final bgLayer = await loadParallaxLayer(
+      ParallaxImageData('parallax/bg.png'),
+    );
+    final mountainFarLayer = await loadParallaxLayer(
+      ParallaxImageData('parallax/mountain-far.png'),
+      velocityMultiplier: Vector2(1.8, 0),
+    );
+    final mountainLayer = await loadParallaxLayer(
+      ParallaxImageData('parallax/mountains.png'),
+      velocityMultiplier: Vector2(2.8, 0),
+    );
+    final treeLayer = await loadParallaxLayer(
+      ParallaxImageData('parallax/trees.png'),
+      velocityMultiplier: Vector2(3.8, 0),
+    );
+    final foregroundTreesLayer = await loadParallaxLayer(
+      ParallaxImageData('parallax/foreground-trees.png'),
+      velocityMultiplier: Vector2(4.8, 0),
+    );
+    final airplaneLayer = await loadParallaxLayer(
+      ParallaxAnimationData(
+        'parallax/airplane.png',
+        SpriteAnimationData.sequenced(
+          amount: 4,
+          stepTime: 0.2,
+          textureSize: Vector2(320, 160),
+        ),
+      ),
+      repeat: planeRepeat,
+      velocityMultiplier: planeSpeed,
+      fill: planeFill,
+      alignment: planeAlignment,
+    );
+
+    final parallax = Parallax(
+      [
+        bgLayer,
+        mountainFarLayer,
+        mountainLayer,
+        treeLayer,
+        foregroundTreesLayer,
+        airplaneLayer,
+      ],
+      baseVelocity: Vector2(20, 0),
```

Comment:
I never imagined a parallax where things have different velocities (other than different speeds). I think it's neat we support it regardless, but I am trying to imagine some use cases

Reply:
I think this is a pretty common thing, having different velocities increases the depth feeling

Reply:
yeah I mean on different directions. rather than just different speeds on the same direction

Reply:
It is not very realistic, but you can make some pretty neat effects with it!
Like when changing directions of the parallax you can have the layers in the back lag behind a bit until they change and it gets this rubber string feeling, if you understand what I mean. :smile:

Reply:
oh that would be super cool!